Features/PrefixCommand.cs

```diff
@@ -0,0 +1,36 @@
+using DSharpPlus.Commands;
+using DSharpPlus.Entities;
+using DSharpPlus.Commands.ContextChecks;
+using ArtcordAdminBot.Features.Helpers;
+
+
+namespace ArtcordAdminBot.Features
+{
+    public class PrefixCommand
+    {
+        private readonly IDatabaseService _databaseService;
+
+        public PrefixCommand(IDatabaseService databaseService)
+        {
+            _databaseService = databaseService;
+        }
+
+        [Command("setprefix")]
+        [System.ComponentModel.Description("Set the bot's prefix")]
+        //[RequirePermissions(DiscordPermissions.Administrator)]
+        public async Task SetPrefixAsync(CommandContext ctx, string newPrefix)
+        {
+            if (string.IsNullOrWhiteSpace(newPrefix))
+            {
+                await ctx.RespondAsync("Prefix cannot be empty.");
+                return;
+            }
+
+            await _databaseService.SetPrefixAsync(ctx.Guild.Id, newPrefix);
```

Comment:
I see on another file (CustomPrefixResolver.cs) you're case-insensitive about the prefixes - if that is the case, I would actually set the prefix to a lowercase version of what has been submitted:

```suggestion
            newPrefix = newPrefix.ToLower();
            await _databaseService.SetPrefixAsync(ctx.Guild.Id, newPrefix);
```